ffi/capi/src/pluralrules.rs

```diff
@@ -170,10 +169,10 @@ impl From<ICU4XPluralOperands> for PluralOperands {
     }
 }
 
-impl From<ICU4XPluralOperands> for ICU4XCreatePluralOperandsResult {
-    fn from(operands: ICU4XPluralOperands) -> Self {
+impl From<PluralOperands> for ICU4XCreatePluralOperandsResult {
```

Comment:
IMO we should convert PluralOperands to ICU4XPluralOperands and not use a From impl for this final conversion to the error type so that the failure case can be handled